<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MW</class>
 <widget class="QMainWindow" name="MW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LG Info Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>PORT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbPorts"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnReadInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Read Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnSwitch2DL">
        <property name="text">
         <string>Switch To Download Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSwitch2NORMAL">
        <property name="text">
         <string>Switch To Normal Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
